vision/fiducial_detection_and_pose.cc

```diff
@@ -0,0 +1,70 @@
+#include "vision/fiducial_detection_and_pose.hh"
```

Comment:
[nit] The most noble thing is to have libraries be ~1 function per file

This is ok and I am [k]nitting, but it's the same idea you've expressed before about functions with the word "and" for libraries imo

not blocking on that